app/src/shared/components/FormControl/InputError/index.jsx

```diff
@@ -1,19 +1,22 @@
 // @flow
 
 import React from 'react'
+import cx from 'classnames'
+
 import styles from './inputError.pcss'
 
 type Props = {
     eligible?: boolean,
```

Comment:
This is a weird prop name i.e. maybe it could be `hasError`?

Reply:
It's odd but inherited from the shared component so I didn't change it.